snap-css-core/src/optimize/suggest/json-exporter.ts

```diff
@@ -0,0 +1,74 @@
+import Loader from '../../load/loader2'
+import Scanner from '../../load/scanner'
+import {execSync} from 'child_process'
+const postcss = require('postcss')
+const cssVariables = require('postcss-css-variables')
+
+export default class JsonExporter {
+  private static trainingData: [string, string][] = [
+    ['bootstrap', 'bs-'],
+    ['bulma', ''],
+    ['foundation', 'foundation-'],
+    ['material-components-web', 'mdc-'],
+    ['materialize', 'materialize-'],
+    ['mdb', 'materialize-'],
+    ['pure', 'pure-'],
+    ['semantic', ''],
+    ['skeleton', ''],
+    // ['tailwind', ''],
+    ['uikit', 'uk-'],
+  ]
+
+  private static getObject(inputPath: string, prefix = '') {
+    const results: object[] = []
+    const [result, message] =  Loader.loadFromFile(inputPath)
+    // eslint-disable-next-line no-console
+    console.log(message)
```

Comment:
Let's remember to comment console messages before committing

Reply:
it is cause it requires checking if there is no console in the runtime environment